submissions/goodnessOlakanmi/html-forms/Product-page.html

```diff
@@ -12,21 +12,19 @@
 			<li><a href="#">About Us</a></li>
 			<li><a href="#">News</a></li>
 			<li><a href="#">Gallery</a></li>
-			<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Contact-Form-Page.html">Contact Form</a></li>
-			<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Product-page.html">Product Page</a></li>
+			<li><a href="Contact-Form-Page.html">Contact Form</a></li>
```

Comment:
Well done. But this link won't work for the contact page. Something is still missing

Reply:
Oh really? I don't think this is the most recent update also. Because @Ireti203 had told me to reduce the length of the link.

Reply:
Did you try it now? I don't think it will work because contact form page is not in the same folder that the product page. You have to first target the folder name. Kindly test what you have implemented

Reply:
Yes, they are in the same folder. I actually created another Product-page that I saved in the same folder as the contact-page. Which is different from the product page I submitted in the previous task.